core/src/main/scala/com/pingcap/tispark/examples/TiBatchWriteRandomDataTest.scala

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright 2019 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.pingcap.tispark.examples
+
+import com.pingcap.tispark.{TiBatchWrite, TiDBOptions}
+import org.apache.spark.SparkConf
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.types.{IntegerType, StructField, StructType}
+import org.apache.spark.sql.{Row, SparkSession, TiContext}
+
+object TiBatchWriteRandomDataTest {
+  def main(args: Array[String]): Unit = {
+    if (args.length < 3) {
+      throw new Exception("wrong arguments!")
+    }
+
+    // test RANDOM_DATA 500000
+    val outputDatabase = args(0)
+    val outputTable = args(1)
+    val size = args(2).toInt
+
+    val enableRegionPreSplit = false
+    val regionSplitNumber = None
+
+    // init
+    val start = System.currentTimeMillis()
+    val sparkConf = new SparkConf()
+      .setIfMissing("spark.tispark.write.enable", "true")
+      .setIfMissing("spark.master", "local[*]")
+      .setIfMissing("spark.app.name", getClass.getName)
+      .setIfMissing("spark.sql.extensions", "org.apache.spark.sql.TiExtensions")
+      .setIfMissing("tidb.addr", "127.0.0.1")
+      .setIfMissing("tidb.port", "4000")
+      .setIfMissing("tidb.user", "root")
+      .setIfMissing("tidb.password", "")
+      .setIfMissing("spark.tispark.pd.addresses", "127.0.0.1:2379")
+
+    val spark = SparkSession.builder.config(sparkConf).getOrCreate()
+    val ti = new TiContext(spark)
+
+    // select
+    spark.sql("show databases").show()
+    spark.sql(s"use $outputDatabase").show()
+    spark.sql("show tables").show()
+
+    val rdd: RDD[Row] = spark.sparkContext.makeRDD(1 to size, 12).map { i =>
+      Row(i)
+    }
+
+    val schema = StructType(
+      List(
+        StructField("i", IntegerType)
+      )
+    )
+
+    var df = spark.sqlContext.createDataFrame(rdd, schema)
+
+    for (i <- 1 to 184) {
+      import org.apache.spark.sql.functions._
+      df = df.withColumn(s"c$i", rand())
+    }
+
+    // batch write
+    val options = new TiDBOptions(
+      sparkConf.getAll.toMap ++ Map("database" -> outputDatabase, "table" -> outputTable)
+    )
+    TiBatchWrite.writeToTiDB(
+      df,
+      ti,
+      options,
+      regionSplitNumber,
+      enableRegionPreSplit
+    )
+
+    // time
+    val end = System.currentTimeMillis()
+    val seconds = (end - start) / 1000
+    println(s"total time: $seconds seconds")
+  }
+}
+
+/*
+CREATE TABLE RANDOM_DATA (
```

Comment:
what is this used for?

Reply:
use this script to create the table before run `TiBatchWriteRandomDataTest`